networkit/cpp/generators/test/ErdosRenyiEnumeratorGTest.cpp

```diff
@@ -82,7 +81,7 @@ static void testEre(const bool directed, const node n, const double prob) {
         } else {
             // count that we produced a number of edges, near the expected number
             // we could apply much sharper bounds here if we allow for rare errors
-            ASSERT_NEAR(num_edges, ere.expectedNumberOfEdges(), 0.6 * ere.expectedNumberOfEdges());
+            ASSERT_NEAR(num_edges, ere.expectedNumberOfEdges(), 0.3 * ere.expectedNumberOfEdges());
```

Comment:
Even with scaling the number of nodes, is it still necessary to worsen the bounds here?